content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -219,7 +219,28 @@ This example demonstrates how to restrict the topology of provisioned volumes to
 zones and should be used as a replacement for the `zone` and `zones` parameters for the
 supported plugins.
 
-{{% code_sample language="yaml" file="storage/storageclass/storageclass-topology.yaml" %}}
```

Comment:
For this, I think we should change the file `storage/storageclass/storageclass-topology.yaml` (rather than convert this to a plain code sample without the ability to copy to clipboard, etc).